src/Components/ShortcutKeyModal/SearchModal.module.css

```diff
@@ -0,0 +1,65 @@
+.modal {
```

Comment:
we should scope the css, so every element in the search modal should be inside of a div called "search-modal",

that way we dont accidentally apply this css to other components in clark with the same class name

an example of this issue, and resolving this issue with scoping is in the below pr

https://github.com/SCE-Development/Clark/pull/1256/files

Reply:
can the class name be shortcut-search-modal, 

modal is kind of generic, we might use this class name somewhere else on accident